prisma-fmt/src/text_document_completion.rs

```diff
@@ -66,12 +97,27 @@ fn push_ast_completions(
                 completion_list.items.push(CompletionItem {
                     label: referential_action.as_str().to_owned(),
                     kind: Some(CompletionItemKind::ENUM),
-                    // what is the difference between detail and documentation?
                     detail: Some(referential_action.documentation().to_owned()),
                     ..Default::default()
                 });
             }
         }
+
+        ast::SchemaPosition::DataSource(_source_id, ast::SourcePosition::Source) => {
+            if connector.has_capability(psl::datamodel_connector::ConnectorCapability::MultiSchema)
+                && schemas_span.is_none()
+                && preview_features.contains(PreviewFeature::MultiSchema)
+            {
+                completion_list.items.push(CompletionItem {
+                    label: "schemas".to_owned(),
+                    insert_text: Some("schemas = []".to_owned()),
+                    kind: Some(CompletionItemKind::PROPERTY),
+                    documentation: Some(Documentation::String("The list of database schemas.".to_owned())),
```

Comment:
Do we have a standardised definition / documentation that we want to use here?

Reply:
Per convo on [slack](https://prisma-company.slack.com/archives/C0350LQSUF2/p1671804341983409), already written documentation is g2g, waiting on [pris.ly](https://github.com/prisma/pris.ly/pull/84) to add docs link